librz/analysis/analysis_private.h

```diff
@@ -0,0 +1,11 @@
+// SPDX-FileCopyrightText: 2023 billow <billow.fun@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#ifndef RZ_ANALYSIS_PRIVATE_H
+#define RZ_ANALYSIS_PRIVATE_H
+
+#include <rz_analysis.h>
+
+RZ_IPI RZ_BORROW RzAnalysisVar *rz_analysis_function_add_var_dwarf(RzAnalysisFunction *fcn, RZ_OWN RzAnalysisVar *var, int size);
```

Comment:
I also think, maybe it should be more generic, not only DWARF, but also PDB? Just use `_dbginfo()` then?

Reply:
But for now it is only used in dwarf, and it may not be used in future pdb-related code.

Reply:
ok